R/translate-sql-base.r

```diff
@@ -273,3 +273,49 @@ base_no_win <- sql_translator(
   lag          = win_absent("lag"),
   order_by     = win_absent("order_by")
 )
+
+#' @export
+#' @rdname sql_variant
+#' @format NULL
+base_odbc_scalar <- sql_translator(
+  .parent = base_scalar,
+  as.numeric    = sql_cast("DOUBLE"),
+  as.double     = sql_cast("DOUBLE"),
+  as.integer    = sql_cast("INT"),
+  as.logical    = sql_cast("BOOLEAN"),
+  as.character  = sql_cast("STRING"),
+  as.Date       = sql_cast("DATE"),
+  round         = function(x, digits = 0L)
```

Comment:
It gives me an `unexpected symbol` error when I add the brackets, it also looks like the original `base_scalar` does not have them.  Maybe I'm missing something here?